installing_on_kubernetes/_topics/migrate_from_appliance.md

```diff
@@ -57,53 +57,48 @@
 
         $ oc edit secret app-secrets
 
-4. Find the orchestrator pod and start a debug session into it. Keep this running in the background...
-
-        $ oc get pods -o name | grep orchestrator
-        $ oc debug pod/orchestrator-123456abcd-789ef
-
-5. Temporarily prevent the orchestrator from starting by adding the following to the deployment:
+4. Temporarily hikack the orchestrator by adding the following to the deployment:
 
         $ oc edit deployment/orchestrator
 
         spec:
           template:
             spec:
-              nodeSelector:
-                kubernetes.io/hostname: nope
-
-6. Delete the old replica set, the new one will sit in "pending" state.
+              containers:
+              - ### Edit the first container and add the following
+                command:
+                - sleep
+                - 1d
 
-        $ oc delete replicaset.apps/orchestrator-123456abcd
+5. Shell into the orchestrator container:
 
-7. Back in the debug pod from step 4:
+        $ oc rsh deploy/orchestrator
 
         $ cd /var/www/miq/vmdb
         $ source ./container_env
         $ DISABLE_DATABASE_ENVIRONMENT_CHECK=1 rake db:drop db:create
 
-8. `oc rsh` into the database pod and restore the database backup
+6. `oc rsh` into the database pod and restore the database backup
 
         $ cd /var/lib/pgsql
         # --- download your backup here ---
-        $ pg_restore -d vmdb_production <your_backup_file>
+        $ pg_restore -d vmdb_production <your_backup_file> # Add -v to see progress
         $ rm -f <your_backup_file>
         $ exit
 
-9. Back in the debug pod from step 4:
+7. Back in orchestrator pod from step 5:
 
         $ rake db:migrate
+        $ rails r "MiqWorker.destroy_all"
```

Comment:
@bdunne What is this step for?  Aside, I thought this happened automatically on startup anyway?  (cc @agrare didn't we just talk about this?)

Reply:
On evm_server startup we kill all workers: https://github.com/ManageIQ/manageiq/blob/master/lib/workers/evm_server.rb#L22

Reply:
Oh I just realized not on morphy though?

Reply:
If we don't kill the workers nicely in morphy, then I'd rather do the following for this one-off case as opposed to the changes in #205

```
rails r "MiqWorker; MiqWorker.inheritance_column = "__type_disabled"; MiqWorker.destroy_all"
```

Reply:
I believe we still did it was just moved out of the rake task and into `lib/workers/evm_server.rb`: https://github.com/ManageIQ/manageiq/blob/morphy/lib/tasks/evm_application.rb#L21

Reply:
Oh yeah you're right, and the delete code exists too https://github.com/ManageIQ/manageiq/blob/morphy/app/models/miq_server/worker_management/monitor/kill.rb#L13-L27